### ISSUE TITLE
Billede-funktionalitet

### ISSUE THREAD

**Issue by Athas** [svip/tabloid#2]
Bag overskriften skal være et tilfældigt valgt billede fra nyhedshjemmesiderne.  (Caching er nok en god idé for ikke at gøre Ekstrabladsandrésen sur.)

**Comment by svip** [comment]
Ja, at huse dem lokalt vil nok være den rigtige løsning.  Forhåbentligt er det ikke så tung en operation.  Det skal i hvert fald kunne paralleliseres.

Men hver gang at overskrifterne hentes, bliver gemmen af billeder også ryddet.  Og om muligt bliver mange af de samme billeder hentet igen.  (Det kan selvfølgelig være man senere kan lave et fiks på det, men hvor vigtigt er det?)
